doc/setup.md

```diff
@@ -148,13 +148,19 @@ setting `debug_login` to `False`.
 ### Creating an administrative user
 
 By default, no user has permission to perform system administration.
-You can add a user with such permissions by running the following from
-Python:
+You can give such permissions as well as other permissions to an existing user by running the following from your terminal in the `skyportal` directory:
+```
+PYTHONPATH=. python tools/set_user_role.py --username="<username>" --role="<role>"
+```
 
+The `username` argument is the name of the user that we want to set to a role, and the `role` argument is the role to give them. The role is optionnal, and defaults to `Super admin`.
```

Comment:
```suggestion
The role is optional, and defaults to `Super admin`.
```

Reply:
You want me to replace all of it by just the explanations about the role ? (i don't explain the username argument anymore ?). I made a modification that made more sense to me while you confirm this or not.